easybuild/easyconfigs/f/fastqc/fastqc-0.11.2.eb

```diff
@@ -1,27 +0,0 @@
-easyblock = 'EB_Java'
-
-name = 'fastqc'
-version = '0.11.2'
-
-homepage = 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/'
-description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
-
-toolchain = {'name': 'dummy', 'version': ''}
-
-source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/fastqc/']
-sources = ['%(name)s_v%(version)s.zip']
-
-dependencies = [('Java', '1.7.0_60')]
-
-postinstallcmds = ["chmod -R a+rx %(installdir)s/fastqc %(installdir)s/run_fastqc.bat"]
```

Comment:
@boegel - There is no need for this to be executable. It's intended to be a Windows batch file and the `fastqc` executable is inteneded for Linux.

Also, the `fastqc` executable is actually a Perl script and Perl isn't included as a dependency. However, it is just a wrapper for processing command line arguments and calling java, so maybe a hard dependency isn't required.